code/_onclick/observer.dm

```diff
@@ -8,7 +8,7 @@
 			return									// seems legit.
 
 	// Things you might plausibly want to follow
-	if((ismob(A) && A != src) || istype(A,/obj/machinery/bot) || istype(A,/obj/singularity))
+	if((ismob(A) && A != src) || istype(A,/obj/machinery/bot) || istype(A,/obj/singularity) || istype(A,/obj/mecha))
```

Comment:
Orbiting as a ghost on Paradise would be hilarious, considering it shows what we looked like when we died/ our selected character in the character setup option should you observe.